ui/DesignSystem/Component/List.svelte

```diff
@@ -10,6 +10,7 @@
 
   export let dataCy = "";
   export let style = "";
+  export let key = "";
```

Comment:
```suggestion
  export let key: string | null = null;
```

This should express optionality not default. The caller should be empowered to determine if the items are keyed or not.